src/main/java/uk/gov/hmcts/reform/hmc/api/config/ServiceBusConfiguration.java

```diff
@@ -89,8 +92,17 @@ public CompletableFuture<Void> onMessageAsync(IMessage message) {
                         String messageReceived =
                                 mapper.writeValueAsString(
                                         mapper.readValue(body.get(0), Hearing.class));
-                        log.debug(messageReceived);
                         Hearing hearing = mapper.readValue(body.get(0), Hearing.class);
```

Comment:
@Profile("!test"). why we use this here?

Reply:
so it wont be considered for bean creation